redshift_sqlalchemy/dialect.py

```diff
@@ -249,6 +250,20 @@ def _get_column_info(self, *args, **kwargs):
 
         return column_info
 
+    def create_connect_args(self, *args, **kwargs):
+        default_args = {
+            'sslmode': 'verify-full',
```

Comment:
I'll be sticking with "verify-full" unless someone reports issues with it, because the documentation on that page that's not wrong is dangerous.

Under the configuration here: https://docs.aws.amazon.com/redshift/latest/mgmt/connecting-ssl-support.html clients will accept certificates minted by Amazon for any Internet server, and anyone with _any_ valid certificate will be able to MITM connections to the redshift server.

If you're using "verify-ca" with your system ca-store your connection can be easily compromised with any free x509 certificate.